OboeTester, CPU LOAD crashes if exited before running

Android version(s): UDC
Android device(s): P7 Pro
App name used for testing: OboeTester
 
**Short description**
New CPU LOAD test crashes if you immediately back out without running.

**Steps to reproduce**
1. Launch OboeTester
1. Press EXTRAS
2. Press CPU LOAD
3. Swipe back or hit BACK key

EXPECT - go back to EXTRAS screen

ACTUAL - crash back to main screen

```
08-09 16:09:57.388 27675 27675 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke direct method 'void com.mobileer.oboetester.DynamicWorkloadActivity$WorkloadUpdateThread.stop()' on a null object reference
08-09 16:09:57.388 27675 27675 E AndroidRuntime: 	at com.mobileer.oboetester.DynamicWorkloadActivity$WorkloadUpdateThread.access$2200(DynamicWorkloadActivity.java:58)
08-09 16:09:57.388 27675 27675 E AndroidRuntime: 	at com.mobileer.oboetester.DynamicWorkloadActivity.onStopTest(DynamicWorkloadActivity.java:354)
08-09 16:09:57.388 27675 27675 E AndroidRuntime: 	at com.mobileer.oboetester.TestAudioActivity.onStop(TestAudioActivity.java:300)
08-09 16:09:57.388 27675 27675 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnStop(Instrumentation.java:1623)
0
```

--- a/apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java
+++ b/apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java
@@ -351,7 +351,10 @@ public class DynamicWorkloadActivity extends TestOutputActivityBase {
 
     @Override
     public void onStopTest() {
-        mUpdateThread.stop();
+        WorkloadUpdateThread updateThread = mUpdateThread;
+        if (updateThread != null) {
+            updateThread.stop();
+        }
         updateButtons(false);
         super.onStopTest();
     }

